ibrahim/styles.css

```
@@ -0,0 +1,270 @@
+  font-family: "Comfortaa", sans-serif;
+  font-size: 36px;
+}
+
+h2 {
+  color: #015759;
+  font-family: "Comfortaa", sans-serif;
+  font-size: 18px;
+}
+
+/* Navigation */
+nav {
+  background-color: #1e1e1e;
+  text-align: center;
+}
+
+nav li {
+  display: inline-block;
+  margin-left: -3px;
+}
+
+nav .first-item {
+  display: inline-block;
+  margin-left: 0px;
+}
+
+nav a {
+  display: block;
+  background-color: #1e1e1e;
+  color: white;
+  padding: 14px 26px;
+  text-decoration: none;
+  font-family: "Raleway", sans-serif;
+  font-size: 14px;
+  font-weight: 700;
+  transition: all 250ms;
+}
+
+nav a:hover,
+nav a:focus {
+  background-color: #015759;
+}
+
+.nav {
+  display: block;
+}
+
+.nav_l {
+  display: none;
+}
+
+/* MaxG Logo */
+.MaxG {
+  height: 30px;
+  width: auto;
+  display: block;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.MaxG_l {
+  height: 60px;
+  width: auto;
+  display: none;
+  margin-top: 50px;
+  margin-bottom: 50px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+/* Art List */
+
+.art-list {
+  margin: 0px -4px 0px -4px;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  align-content: center;
+}
+
+figure {
+  padding: 0px 4px 0px 4px;
+  position: relative;
+}
+
+figure img {
+  padding: 0px 0px 4px 0px;
+}
+
+figcaption {
+  background-color: hsla(0, 0%, 12%, 0.75);
+  color: white;
+  position: absolute;
+  bottom: 13px;
+  right: 9px;
+  padding: 6px;
+  text-align: center;
+  font-size: 12px;
+  font-weight: 800;
+}
+
+.main-content {
+  padding-bottom: 20px;
+}
+
+/* BOPA logo */
+.bmore {
+  height: auto;
+  width: 100%;
+  display: block;
+  margin-right: auto;
+  margin-left: auto;
+  margin-top: 10px;
+}
+
+.footer {
+  border-top: 0.25px silver solid;
+  margin-left: 10px;
+  margin-right: 10px;
+  padding-top: 5px;
+}
+
+.footer-text {
+  margin: 15px 15px;
+}
+
+/* Responsive */
+.row {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 0;
+}
+
+/* Tablet */
+@media screen and (min-width: 360px) {
+  .nav {
+    display: none;
+  }
+
+  .nav_l {
+    display: block;
+  }
+
+  .MaxG {
+    display: none;
+  }
+
+  .MaxG_l {
+    display: block;
+    height: 45px;
+    margin-top: 35px;
+    margin-bottom: 35px;
+  }
+
+  .art-list {
+    /*     min-height: 500px;
+    max-height: 900px; */
+    max-height: 135vw;
+  }
+
+  .art-list li {
+    flex-grow: 0;
+    flex-shrink: 1;
+    flex-basis: auto;
+    width: 33.333%;
+  }
+
+  body {
+    margin: 0;
+    font-size: 14px;
+  }
+
+  h2 {
+    font-size: 22px;
+  }
+
+  .text {
+    margin: 30px 40px;
+  }
+
+  .footer-text {
+    margin: 30px 30px;
+  }
+
+  .bmore {
+    height: auto;
+    width: 300px;
+    display: block;
+    margin-right: auto;
+    margin-left: -14px;
+    margin-top: 10px;
+  }
+}
+
+/* Desktop */
+@media screen and (min-width: 768px) {
+  .art-list {
+    max-height: 125vw;
+  }
+
+  body {
+    margin: 0px 10% 0px 10%;
```

Since before the `margin` was set to zero, here I would only update what's changing, like so:
```suggestion
    margin-left: 10%;
    margin-right: 10%;
```